rust/worker/src/blockstore/arrow/types.rs

```diff
@@ -26,7 +26,7 @@ pub(crate) trait ArrowWriteableValue: Value {
     fn get_delta_builder() -> BlockStorage;
 }
 
-pub(crate) trait ArrowReadableKey<'referred_data>: Key + PartialEq {
+pub(crate) trait ArrowReadableKey<'referred_data>: Key + PartialOrd {
```

Comment:
Why this change?

Reply:
This was left over from testing. Originally I was comparing just the key (and not the prefix). Later I switched to comparing the composite key so this is not needed. Thanks for flagging this!